css/metabox-primary-category.css

```diff
@@ -0,0 +1,32 @@
+.wpseo-primary-term > label,
+.wpseo-is-primary-term {
+	font-weight: bold;
+}
+
+.wpseo-primary-term > .wpseo-make-primary-term,
+.wpseo-term-unchecked > .wpseo-make-primary-term {
+	display: none;
```

Comment:
The `.wpseo-make-primary-term` link (button) above has spans for screen readers, but I don't think those won't be viewable by screen readers since the parent is display none.